src/main/groovy/com/bmuschko/gradle/docker/tasks/network/DockerRemoveNetwork.groovy

```diff
@@ -0,0 +1,10 @@
+package com.bmuschko.gradle.docker.tasks.network
+
+class DockerRemoveNetwork extends DockerExistingNetwork {
+    @Override
+    void runRemoteCommand(final dockerClient) {
+        final networkId = getNetworkId()
+        logger.quiet "Removing network '$networkId'."
+        dockerClient.removeNetworkCmd(networkId).exec()
```

Comment:
Yep, this one is https://github.com/docker-java/docker-java/blob/f6d225d2c2e65aa058da5b94c2350cbe5d5f96c9/src/main/java/com/github/dockerjava/api/command/RemoveNetworkCmd.java#L14